### DISCUSSION TITLE
Request for easier way to debug

### DISCUSSION THREAD

**Discussion by caherrerapa** [excid3/noticed#507] [Ideas]
This gem is amazing, it helps us to clean up our architecture.

I only have a feature request, it's a bit hard to currently debug. In our use case the noticed_events record is always created but the noticed_notifications silently fail to be created, it's hard to know how or where it is failing

**Comment by excid3** [comment]
You can `bundle open noticed` and put a `binding.irb` in the code to see what's going on.
